docs/cep/stream-workers-example.md

```diff
@@ -629,7 +631,7 @@ The following example uses the code snippets provided in this tutorial.
             @info(name='Dump')
             INSERT INTO SampleCargoAppDestTable
             SELECT weight
-            FROM SampleCargoAppInputTable;
+            FROM SampleCargoAppInputTable;`
```

Comment:
Is the addition here intentional? It does not look correct.

Reply:
Yes it's intentional

Reply:
There was a missing backtick here, the ending backtick for line 583/585 is missing
const updatedAppDefinition = `

The ending backtick should be there in line 634 as added

Reply:
Nice catch @koshy-thomas :+1: